src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -308,16 +317,17 @@ def cancel(self) -> bool:
                                 _LOCAL_CANCELLATION_DETAILS, None, None))
 
     def __await__(self) -> ResponseType:
-        """Wait till the ongoing RPC request finishes.
-
-        Returns:
-          Response of the RPC call.
-
-        Raises:
-          RpcError: Indicating that the RPC terminated with non-OK status.
-          asyncio.CancelledError: Indicating that the RPC was canceled.
-        """
-        response = yield from self._call
+        """Wait till the ongoing RPC request finishes."""
+        try:
+            response = yield from self._call
+        except asyncio.CancelledError:
+            # Even if we converted all other CancelledError, there is still
+            # this corner case. If the application cancels immediately after
+            # the Call object is created, we will observe this
+            # `CancelledError`.
```

Comment:
does this comment apply yet ^^?

Reply:
Yes. This statement is still true that if we removed this catch clause, test case `test_cancel_unary_unary_in_task` will fail.

Changed `converted` to `caught`.